bazel/external/wee8.genrule_cmd

```diff
@@ -7,6 +7,18 @@ ARCH="$$(uname -m)"
 SYSTEM="$$(uname -s)"
 PLATFORM="$${SYSTEM}-$${ARCH}"
 
+detect::compiler() {
+  if $${CXX} --version 2>&1 | grep -q clang ; then
+    echo clang
+  fi
+
+  if $${CXX} --version 2>&1 | grep -q gcc ; then
+    echo gcc
+  fi
+
+  echo unknown
```

Comment:
There is no early return, so I think that `unknown` is always emitted, hence the build failure.

Reply:
Ugh, what was I thinking :-/

Reply:
Ping?